tests/fast-integration/eventing-test/README.md

```diff
@@ -28,14 +29,35 @@ COMPASS_CLIENT_SECRET=       # Only required if KYMA_TYPE=SKR
 COMPASS_TENANT=              # Only required if KYMA_TYPE=SKR
 GARDENER_KUBECONFIG=         # Only required if KYMA_TYPE=SKR
 ```
+>**IMPORTANT:** The `TEST_SUFFIX` is required for every test run for SKR cluster. It needs to be 4 characters long, as it is the name for the compass scenario.
+> The eventing tests are adding a scenario with the `"test-TEST_SUFFIX"` name and if you did not clean the compass scenario (which is done in the `eventing-test-cleanup.js` script),
+> the tests will try to reuse an already existing scenario and will fail on the tests preparation with the following error: *Update of API/Event is not supported yet for the compass scenario*.
+
 >**NOTE:** The Event Mesh Credentials JSON file can be downloaded from the BTP Cockpit under your subaccount instances.
 
 3. Run the following command to set up the environment variables in your system:
 ```bash
 export $(xargs < .env)
 ```
 
-4.  Execute the Eventing tests:
+4. Execute the Eventing tests locally
+
+- **inside the OSS cluster:**
 ```bash
 npm run test-eventing
-```
\ No newline at end of file
+```
+- **inside the SKR cluster:**
+```bash
+npm run test-eventing TEST_SUFFIX=abcd
+```
+
+## Troubleshooting ##
+
+1. if the `TEST_SUFFIX` env variable was not set during the test execution or a scenario with that suffix already exists, you can get the following error:
```

Comment:
"can" = maybe? or definitely? If definitely, then drop it

Why the numbering? Where is number 2 ?

```suggestion
If the `TEST_SUFFIX` environment variable was not set during the test execution or a scenario with that suffix already exists, you get the following error:
```

Reply:
done